### DISCUSSION TITLE
[CHORE] Is there an example of Elsa 3.0 implementing various activities in designer mode? Thank you.

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5628] [Q&A]
Is there an example of Elsa 3.0 implementing various activities in designer mode? Thank you.

**Comment by glime-ai** [comment]
As of my last update, there isn't a specific example provided in the official Elsa 3.0 documentation or GitHub repository that showcases the implementation of various activities in designer mode. However, you can refer to the general documentation and resources available for Elsa 3.0 to understand how to work with the visual designer and create workflows.

The visual designer in Elsa 3.0 has been revamped and includes features like drag & drop, multi-select, undo, redo, copy & paste, and more. While the documentation is still a work in progress, you can explore the Elsa Dashboard, which is a part of the Elsa Workflows suite, to visually design workflows.

For more detailed examples and guidance, you may want to follow the GitHub repository for updates, check out discussions on forums like Stack Overflow, or reach out to the community for shared experiences and examples.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
